omero/users/cli/export.rst

```diff
@@ -0,0 +1,38 @@
+Export images
+-------------
+
+The |CLI| export command allows you to export data in XML and TIFF formats from
```

Comment:
We use OME-TIFF in the format docs so should be consistent with that here please